packages/oft-evm/contracts/NativeOFTAdapter.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import { MessagingFee, MessagingReceipt, OFTCore, OFTReceipt, SendParam } from "./OFTCore.sol";
+
+/**
+ * @title NativeOFTAdapter Contract
+ * @dev NativeOFTAdapter is a contract that adapts native currency to the OFT functionality.
+ *
+ * @dev WARNING: ONLY 1 of these should exist for a given global mesh,
+ * unless you make a NON-default implementation of OFT and needs to be done very carefully.
+ * @dev WARNING: The default NativeOFTAdapter implementation assumes LOSSLESS transfers, ie. 1 native in, 1 native out.
+ */
+abstract contract NativeOFTAdapter is OFTCore {
+
+    address public constant NATIVE_TOKEN_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
```

Comment:
I will defer to @TRileySchwarz here, but we pretty much adapted the non-standard convention of using `address(0)` instead of this.  I am not quite sure what we want to support going forward, but historically we used `address(0)` as it is chain agnostic.

Reply:
Ah I see, makes sense! @TRileySchwarz I will change this to return `address(0)` based on @ryandgoulding's recommendation above, please lmk if you disagree